### DISCUSSION TITLE
how to use sqlite in hololens

### DISCUSSION THREAD

**Discussion by dahaideyu** [microsoft/MixedRealityToolkit-Unity#9680] [Q&A] [answered]
how to use sqlite in hololens2

**Comment by polar-kev** [comment, marked as answer]
Hi @dahaideyu, this sounds like a good question for the Unity forums, https://forum.unity.com/.
Here are some instructions on how to use SQLite in a UWP but this does not include any information about Unity, https://docs.microsoft.com/en-us/windows/uwp/data-access/sqlite-databases. 

Searching for Unity and SQLite might also help.

**Reply by dahaideyu** [reply]
yes,but no success. HoloLens2 need ARM64  dll. when you set your app install Hololens2,  it will be  a issue: no dll error.
